js/src/env/factory.ts

```diff
@@ -27,7 +29,14 @@ export class WorkspaceFactory {
         if (this.workspace) {
             return;
         }
-        logger.debug(`Creating workspace with env=${this.workspaceConfig.env} and kwargs=${JSON.stringify(this.workspaceConfig.config)}`);
+
+        const sanitizedConfig = {
+            ...this.workspaceConfig,
+            host: this.workspaceConfig.config.composioBaseURL,
+            composioAPIKey: this.workspaceConfig.config.composioAPIKey ? "REDACTED" : "NOT DEFINED"
```

Comment:
Chore: Let's use constant values for "REDACTED" and "NOT DEFINED". It would be easy to reuse and change in future, if required